src/components/Contacts/ContactDetails/ContactDetailsHeader/DeleteContactModal/DeleteContactModal.tsx

```diff
@@ -39,9 +98,14 @@ export const DeleteContactModal: React.FC<DeleteContactModalProps> = ({
     >
       <DialogContent dividers>
         <DialogContentText>
-          {t(
-            'Are you sure you want to permanently delete this contact? Doing so will permanently delete this contacts information, as well as task history. This cannot be undone. If you wish to keep this information, you can try hiding this contact instead.',
-          )}
+          {canDelete &&
+            t(
+              'Are you sure you want to permanently delete this contact? Doing so will permanently delete this contacts information, as well as task history. This cannot be undone. If you wish to keep this information, you can try hiding this contact instead.',
+            )}
+          {!canDelete &&
+            t(
+              "This contact cannot be deleted because part or all of the contact's data is sourced from a third Party. Please email Donation Services to request that this contact be deleted, or you can hide this contact instead.",
```

Comment:
What do you think of this change to match more closely the text in `EditContactAddressModal`?

```suggestion
              "This contact cannot be deleted because part or all of the contact's data syncs with Donation Services. Please email Donation Services to request that this contact be deleted, or you can hide this contact instead.",
```